app/models/user.rb

```diff
@@ -1,3 +1,8 @@
 class User < ActiveRecord::Base
   has_secure_password
+  has_many :orders
+
+  def full_name
```

Comment:
Good thinking Spencer. This will make our lives easier when we need to reference the user with first and last names.